packages/ui/src/components/filter/components/filter-buttons/filter-buttons.styles.ts

```diff
@@ -1,5 +1,19 @@
 import { tv } from 'tailwind-variants';
 
 export const styles = tv({
-  base: 'flex w-full gap-1 overflow-auto whitespace-nowrap',
+  slots: {
+    container: 'relative flex justify-start',
+    scrollButtonLeft:
+      'absolute h-5 resize-none rounded-none border-none border-l-white transition-all duration-100 contrast-more:bg-black left-0 bg-[linear-gradient(90deg,_#fff_66%,_transparent)] hover:bg-[linear-gradient(90deg,_#fff_66%,_transparent)]',
+    scrollButtonLeftHidden:
+      'absolute h-5 resize-none rounded-none border-none border-l-white transition-all duration-100 contrast-more:bg-black left-0 bg-[linear-gradient(90deg,_#fff_66%,_transparent)] hover:bg-[linear-gradient(90deg,_#fff_66%,_transparent)] invisible',
+    scrollButtonRight:
+      'absolute h-5 resize-none rounded-none border-none border-l-white transition-all duration-100 contrast-more:bg-black right-0 bg-[linear-gradient(270deg,_#fff_66%,_transparent)] hover:bg-[linear-gradient(270deg,_#fff_66%,_transparent)]',
+    scrollButtonRightHidden:
+      'absolute h-5 resize-none rounded-none border-none border-l-white transition-all duration-100 contrast-more:bg-black right-0 bg-[linear-gradient(270deg,_#fff_66%,_transparent)] hover:bg-[linear-gradient(270deg,_#fff_66%,_transparent)] invisible',
```

Comment:
Refactor to use [compoundSlots](https://www.tailwind-variants.org/docs/slots) instead of repeating styles